www/src/data/sidebars/doc-links.yaml

```
@@ -490,6 +490,8 @@
           link: /docs/migrating-from-v1-to-v2/
         - title: Migrating from v0 to v1
           link: /docs/migrating-from-v0-to-v1/
+        - title: Upgrade Gatsby for minor or patch releases
+          link: /docs/upgrade-gatsby-and-dependencies/
```

The breadcrumb and other components are slated for our contributing docs improvements as part of https://github.com/gatsbyjs/gatsby/issues/11946, but here's how to add a `breadcrumbTitle` in the meantime. This feature will show a shorter piece of text in a breadcrumb, useful for brevity purposes when positioned next to its parent title (usually making some of the words redundant):

```suggestion
          link: /docs/upgrade-gatsby-and-dependencies/
          breadcrumbTitle: Upgrade for Minor or Patch Releases
```